package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "proxay",
-  "version": "1.6.1",
+  "version": "1.6.2",
```

Comment:
**Issue**: Please remove the version bump from this PR. We will do a release later on that contains this changeset as part of the release.

Reply:
Done